source/manuals/logit-io.html.md.erb

```diff
@@ -1,22 +1,22 @@
 ---
-title: Sending logs securely to Logit.io
+title: How to send logs securely to Logit
 ---
 
 # <%= current_page.data.title %>
 
-Logit offers logstash endpoints. However in their default mode there is no authentication on them, this means anyone who has the logstash address could inject fake logs into your stash.
+*more introduction here*
```

Comment:
Is this unfinished? if so, could you mark your PR as work in progress so people can review it in that context?